default/stringtables/en.txt

```diff
@@ -1147,6 +1147,18 @@ Gas Giants Habitable size
 RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
+RULE_THRESHOLD_HUMAN_PLAYER_WIN
+Limit human player winners
```

Comment:
limit -> max, mention allied

Reply:
With `RULE_ONLY_ALLIANCE_WIN` disabled winners players are allowed not be allied.

Reply:
Changed without alliance mentioned.